transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -211,14 +164,13 @@ NVTE_Fused_Attn_Backend nvte_get_fused_attn_backend(
  *
  * Support Matrix:
    \verbatim
-   | backend | precision |    qkv layout   |       bias         |                 mask                  | dropout | sequence length | head_dim |
-   | 0       | FP16/BF16 | QKV_INTERLEAVED | NO/POST_SCALE_BIAS | NO_MASK/PADDING/CAUSAL/PADDING_CAUSAL |   Yes   |     <= 512      |    64    |
-   | 1       | FP16/BF16 | QKV_INTERLEAVED |       NO_BIAS      |     PADDING/CAUSAL/PADDING_CAUSAL     |   Yes   |      > 512      |  64, 128 |
-   | 2       | FP8       | QKV_INTERLEAVED |      NO_BIAS       |            PADDING_MASK               |   Yes   |     <= 512      |    64    |
+   | backend | precision | qkv layout |       bias         |                  mask                 | dropout | sequence length | head_dim |
+   | 0       | FP16/BF16 | 3HD or H3D | NO/POST_SCALE_BIAS | NO/PADDING/CAUSAL/PADDING_CAUSAL_MASK |   Yes   |     <= 512      |    64    |
+   | 1       | FP16/BF16 | 3HD or H3D |      NO_BIAS       |          NO/PADDING/CAUSAL_MASK       |   Yes   |      > 512      |   <= 128 |
```

Comment:
Doesn't the arbitrary support all mask? The seqlen may be adjust to unlimited or `%32`. Bias is also supported, right?

Reply:
Fixed some of the columns and prettified it a little bit. Thanks!